src/test/java/com/wikia/webdriver/testcases/auth/ForgottenPasswordTests.java

```diff
@@ -27,39 +26,32 @@ public void anonCanRemindPasswordFromAuthModal() {
     NavigationBar signInLink = new NavigationBar(driver);
     signInLink.clickOnSignIn();
     AuthModal loginModal = new AuthModal();
-    loginModal.clickForgotPasswordLink();
-    SpecialUserLoginPageObject login = new SpecialUserLoginPageObject(driver);
-    login.remindPasswordNewAuth(userName);
-    login.verifyMessageAboutNewPassword(userName);
-    login.clickLogInLink();
-    String
-        newPassword =
-        login.receiveMailWithNewPassword(credentials.email, credentials.emailPassword);
-    loginModal.login(userName, newPassword);
-    loginModal.verifyUserLoggedIn(userName);
+    loginModal
+      .clickForgotPasswordLink()
+      .requestLinkForUsername(userName);
+
+    String resetLink = base.getPasswordResetLink(credentials.email, credentials.emailPassword);
+    driver.get(resetLink);
+    loginModal.isSignInOpened();
+    //loginModal.login(userName, newPassword);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)